dask/dataframe/tests/test_utils_dataframe.py

```diff
@@ -126,13 +126,19 @@ def __next__(self):
     assert len(meta.index) == 0
 
     # Categoricals
-    meta = make_meta({"a": "category"}, parent_meta=df)
+    meta = make_meta({"a": "category"}, parent_meta=df)  # fails
```

Comment:
I think this is safe to remove

```suggestion
    meta = make_meta({"a": "category"}, parent_meta=df)
```

Reply:
Oh yes, thanks for catching it!